src/rgw/logrotate.conf

```diff
@@ -5,7 +5,14 @@
     sharedscripts
     postrotate
         if which service > /dev/null 2>&1 && [ -x `which service` ]; then
-            service ceph-radosgw reload >/dev/null
+            case $(lsb_release -si) in
```

Comment:
@Abhishekvrshny (NIT) Though close, Please consider to use title < 50 characters for PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits